tinygrad/graph.py

```diff
@@ -4,9 +4,10 @@
 except ImportError:
   nx = None # graph won't work
 from collections import defaultdict
-from typing import Dict, List, Optional
-from tinygrad.ops import UnaryOps, BinaryOps, ReduceOps, MovementOps, LoadOps, FusedOps, Op, OpType, LazyOp, get_buffers, get_lazyops
-from tinygrad.lazy import LazyBuffer
+from typing import TYPE_CHECKING, Dict, List, Optional
+from tinygrad.ops import UnaryOps, BinaryOps, ReduceOps, MovementOps, LoadOps, FusedOps, Op, OpType, LazyOp
+if TYPE_CHECKING:
```

Comment:
Is this needed? Does it make it faster?

Reply:
The reason for this is that we used to import the graph utilities in the `__init__` of LazyBuffer

We need this to break the circular dependency and be able to import the graph utils only once at the top-level of `lazy.py`

Reply:
Hmm, I think there has to be cleaner way. The graph utils should sit at the top of the stack, I'm even fine with dynamically importing them only if GRAPH env var is set.

Reply:
We can't do that because the `GRAPH` env var itself is set in `graph.py`. Maybe we could move all env vars to a separate file?

Reply:
oh I'm fine with `GRAPH` env var moving to helpers.